governance/votes/Executive Vote - May 17, 2023.md

```diff
@@ -0,0 +1,88 @@
+* 5.95 MKR will be transferred to [CodeKnight](https://forum.makerdao.com/t/codeknight-constitutional-delegate-communications/20443) at [0xf6006d4cF95d6CB2CD1E24AC215D5BF3bca81e7D](https://etherscan.io/address/0xf6006d4cF95d6CB2CD1E24AC215D5BF3bca81e7D).
+* 5.95 MKR will be transferred to [Flip Flop Flap Delegate LLC](https://forum.makerdao.com/t/flip-flop-flap-delegate-llc-aligned-delegate-recognition-submission-and-communication/20477) at [0x3d9751EFd857662f2B007A881e05CfD1D7833484](https://etherscan.io/address/0x3d9751EFd857662f2B007A881e05CfD1D7833484).
+* 5.95 MKR will be transferred to [PBG](https://forum.makerdao.com/t/pbg-aligned-delegate-communication-platform/20471) at [0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2).
+* 5.95 MKR will be transferred to [UPMaker](https://forum.makerdao.com/t/upmaker-ad-submission/20577) at [0xbB819DF169670DC71A16F58F55956FE642cc6BcD](https://etherscan.io/address/0xbB819DF169670DC71A16F58F55956FE642cc6BcD).
+
+### DAI Budget Streams
+
+As per [MIP107](https://mips.makerdao.com/mips/details/MIP107), the following DAI budget streams will be created if this executive proposal passes:
+
+#### DAI Budget Streams
+
+| Budget | DAI Amount | Start Date | End Date | Destination Address |
+|---|---|---|---|---|
+| Governance Security Engineering | 2,200,000 | 2023-05-01 | 2024-05-01 (--> $check date since it's midnight <---) | [0x569fAD613887ddd8c1815b56A00005BCA7FDa9C0](https://etherscan.io/address/0x569fAD613887ddd8c1815b56A00005BCA7FDa9C0) (--> $source of truth for this address other than spreadsheet? <---) |
+ Multichain Engineering | 2,300,000 | 2023-05-01 | 2024-05-01 (--> $check date since it's midnight <---) | [0x868B44e8191A2574334deB8E7efA38910df941FA](https://etherscan.io/address/0x868B44e8191A2574334deB8E7efA38910df941FA) (--> $source of truth for this address other than spreadsheet? <---)|
+
+Note that these streams are implemented as a [dssvest stream](https://manual.makerdao.com/module-index/module-token-streaming) that is controlled by the Protocol Engineering Facilitators, and paid out to Ecosystem Actors in order to do work necessary for Governance Security Engineering and Multichain Engineering to function properly.
+
+### Data Insights Core Unit (DIN-001) MKR Transfer
+
+As per their successful [MKR vesting proposal](https://mips.makerdao.com/mips/details/MIP40c3SP64#mkr-vesting) **103.16 MKR** (--> $source of truth for MKR amount apart from spreadsheet? <--)will be transferred to the [Data Insights Core Unit (DIN-001)](https://mips.makerdao.com/mips/details/MIP39c2SP22) wallet [0x7327Aed0Ddf75391098e8753512D8aEc8D740a1F](https://etherscan.io/address/0x7327Aed0Ddf75391098e8753512D8aEc8D740a1F), if this executive proposal passes.
```

Comment:
Spreadsheet is source of truth, Tadeo confirmed it, and its less than stated in the MIP.